builtin/game/detached_inventory.lua

```diff
@@ -15,6 +15,6 @@ function core.create_detached_inventory(name, callbacks)
 	end
 	stuff.mod_origin = core.get_current_modname() or "??"
 	core.detached_inventories[name] = stuff
-	return core.create_detached_inventory_raw(name)
+	return core.create_detached_inventory_raw(name, player_name)
```

Comment:
Just a minor thing while you're editing around here: can the raw version be hidden, or is it used anywhere else? Can't grep from phone.